tests/e2escenarios/e2e_test.go

```diff
@@ -93,7 +93,7 @@ var _ = Describe("E2E Test", func() {
 
 			// "running odo list"
 			stdout := helper.Cmd("odo", "list", "component").ShouldPass().Out()
-			helper.MatchAllInOutput(stdout, []string{componentName, "nodejs", "Dev"})
+			helper.MatchAllInOutput(stdout, []string{componentName, "Node.js", "Dev"})
```

Comment:
It looks like this should be left as is. Same for the change on L119.
```
  Expected
      <string>:  â€¢  Listing components from namespace 'e2e-test47grk'  ...
      
 âœ“  Listing components from namespace 'e2e-test47grk' [455ms]
       NAME      PROJECT TYPE  RUNNING IN  MANAGED      
       * nlbvnt  nodejs        Deploy      odo (v3.1.0) 
      
  to contain substring
      <string>: Node.js
  In [It] at: /go/odo_1/tests/helper/helper_generic.go:76
```

Reply:
Thanks. It shows there is an inconsistency for the PROJECT value between Dev and Deploy'ed components. I'll create an issue to follow this.